templates/tutorialv2/labels/view-labels.html

```diff
@@ -0,0 +1,63 @@
+{% extends "base.html" %}
+{% load profile %}
+{% load thumbnail %}
+{% load date %}
+{% load i18n %}
+{% load captureas %}
+{% load times %}
+
+
+{% block title %}
+    {{ headline }}
+{% endblock %}
+
+
+{% block breadcrumb %}
+    <li>{{ headline }}</li>
+{% endblock %}
+
+{% block headline %}
+    {{ headline }}
+{% endblock %}
+
+{% block sidebar %}
+<div class="sidebar">
+    <div class="mobile-menu-bloc mobile-all-links" data-title="Filtrer">
+        <h3>Filtrer</h3>
+        <ul>
+            {% for label in labels %}
+                <li>
+                    <a href="{% url 'content:view-labels' %}?label_{{ label.id }}" class="{% if current_filter_pk == label.pk %}selected{% endif %}">{{ label.name }} ({{ label.num_contents }})</a>
+                </li>
+            {% endfor %}
+        </ul>
+    </div>
+</div>
+{% endblock %}
+
+{% block content %}
+<section class="flexpage-wrapper">
+
+    <p>
+        {% blocktrans %}
+        Cette page est un aperçu de la labellisation. Elle vise à valoriser les publications de manière permanente sur le site
```

Comment:
```suggestion
        Cette page est un aperçu de la labellisation. Elle vise à valoriser les publications de manière permanente sur le site.
```

(mettre la description du label quelque part ici ? attention, les labels n'ont pas obligatoirement une description, mais c'est  une contrainte qu'on peut rajouter)